common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -139,33 +147,43 @@ public static void setEnabled(boolean enabled) {
         setLevel(enabled? Level.SIMPLE : Level.DISABLED);
     }
 
+    private static boolean isImmutableDisabled() {
```

Comment:
Might be slightly better:

```suggestion
    private static boolean isImmutablyDisabled() {
```